etc/scripts/synclic.py

```diff
@@ -892,7 +892,7 @@ def update_external(_attrib, _sc_val, _ext_val):
 
             continue
 
-        if isinstance(scancode_value, basestring) and isinstance(external_value, basestring):
+        if isinstance(scancode_value, (str, bytes)) and isinstance(external_value, (str, bytes)):
```

Comment:
This commit is repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/c7399c4104a14796d5b1a3cce2f265464f977e0f